resources/translations/en.edn

```diff
@@ -218,36 +218,42 @@
-                                       :message-to-handler "Dear %1,\n\n%2 has added terms of use to the application %3 from %4.\n\nView application: %6"}
-          :application-submitted {:subject "Application submitted (%4: %3)"
-                                  :message "Dear %1,\n\n%4 has submitted an application (%3): %5.\n\nView application: %6"}
-          :comment-requested {:subject "Review requested (%2: %3)"
-                              :message "Dear %1,\n\n%2 has requested a review on application %3.\n\nComment: %6"}
-          :commented {:subject "Review added (%2: %3)"
-                      :message "Dear %1,\n\n%2 has reviewed application %3.\n\nView application: %6"}
-          :decided {:subject "Decision made (%2: %3)"
-                    :message "Dear %1,\n\n%2 has made a decision on application %3.\n\nView application: %6"}
-          :decision-requested {:subject "Decision requested (%2: %3)"
-                               :message "Dear %1,\n\n%2 has requested your decision on application %3.\n\nView application: %6"}
-          :footer "\n\nPlease don't reply to this automatically generated message."
+  :email {:application-approved {:subject-to-applicant "Your application has been approved (%3)"
+                                 :message-to-applicant "Dear %1,\n\nYour application %3 has been approved.\n\nYou can review the application and the decision: %6"
+                                 :subject-to-handler "Application approved (%3)"
+                                 :message-to-handler "Dear %1,\n\n%2 has approved the application %3 from %4.\n\nYou can review the application and the decision: %6"}
+          :application-rejected {:subject-to-applicant "Your application has been rejected (%3)"
+                                 :message-to-applicant "Dear %1,\n\nYour application %3 has been rejected.\n\nYou can review the application and the decision: %6"
+                                 :subject-to-handler "Application rejected (%3)"
+                                 :message-to-handler "Dear %1,\n\n%2 has rejected the application %3 from %4.\n\nYou can review the application and the decision: %6"}
+          :application-closed {:subject-to-applicant "Your application has been closed (%3)"
+                               :message-to-applicant "Dear %1,\n\nYour application %3 has been closed.\n\nYou can review the application: %6"
+                               :subject-to-handler "Application closed (%3)"
+                               :message-to-handler "Dear %1,\n\n%2 has closed the application %3 from %4.\n\nYou can review the application: %6"}
+          :application-returned {:subject "Application has been returned for modifications (%3)"
+                                 :message-to-applicant "Dear %1,\n\nYour application %3 has been returned to you for modifications.\n\nYou can modify and re-submit the application: %6"
```

Comment:
re-submit -> resubmit

Reply:
heh, that was verbatim from the THL materials, I'll fix it